renovate.json

```
@@ -11,7 +11,12 @@
 			"automerge": true
 		},
 		{
-			"packagePatterns": ["^@nrlw/", "^@angular", "typescript"],
+			"packagePatterns": [
+				"^@nrlw/",
+				"^@angular",
```

Can we unify the pattern definition? Because the scoping of `nrwl`, `angular` and `nestjs` is similar while the definition is different which is confusing.

```suggestion
				"^@angular/",
```

---

We also want @angular-devkit and @angular-eslint packages to be ignored

---

How about adding them for clarity too?

---

done